packages/debug/rfkill/package.mk

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
We prefer `GPL-2.0` unless there is a deliberate reason for using `GPL-2.0-or-later`

Reply:
no reason for this, just copy&paste